bcda/auth/ssas.go

```diff
@@ -34,9 +35,10 @@ func (s SSASPlugin) ResetSecret(clientID string) (Credentials, error) {
 }
 
 // RevokeSystemCredentials revokes any existing credentials for the given clientID.
-func (s SSASPlugin) RevokeSystemCredentials(clientID string) error {
-	// s.client.DeleteCredentials()
-	return errors.New("Not yet implemented")
+func (s SSASPlugin) RevokeSystemCredentials(ssasID string) error {
+	err := s.client.DeleteCredentials(ssasID)
```

Comment:
Do we want to return the value of the `s.client.DeleteCredentials()` call directly, and make this function into one line?